composer.json

```diff
@@ -12,8 +12,8 @@
         "guzzle/http": "~3.7",
         "jms/serializer": "0.12.*",
         "psr/log": "~1.0",
-        "symfony/console": "~2.3",
-        "symfony/expression-language": "~2.4"
+        "symfony/console": "3.*",
+        "symfony/expression-language": "3.*"
```

Comment:
you should not drop support for the Symfony LTS releases IMO (especially when nothing in the code requires it). you should allow both Symfony 2 and 3 instead